src/models/config.go

```diff
@@ -1,85 +1,179 @@
 package models
 
 import (
+	"encoding/json"
+	"strings"
+
 	local "github.com/fibercrypto/FiberCryptoWallet/src/main"
-	"github.com/fibercrypto/FiberCryptoWallet/src/util/logging"
 
 	"github.com/therecipe/qt/qml"
 
 	qtcore "github.com/therecipe/qt/core"
 )
 
-var logConfigManager = logging.MustGetLogger("modelsConfigManager")
-
-type WalletSource struct {
+type ConfigManager struct {
 	qtcore.QObject
-	_ int    `property:"sourceType"`
-	_ string `property:"source"`
+	configManager *local.ConfigManager
+	_             func()                      `constructor:"init"`
+	_             func(string, string)        `slot:"setValue"`
+	_             func(string) string         `slot:"getValue"`
+	_             func() []string             `slot:"getSections"`
+	_             func(string) *ConfigSection `slot:"getSection"`
 }
```

Comment:
We need a way to get the default values. Something like a `getDefault("skycoin/walletSource/1/Source")` should return `"/home/username/.skycoin/wallets"`.

Reply:
@stdevMauricio1802 already added that feature . can this conversation be marked as resolved ?